src/Makefile.am

```diff
@@ -439,7 +435,9 @@ endif
 if !ENABLE_WALLET
 libbitcoin_node_a_SOURCES += dummywallet.cpp
 endif
+#
```

Comment:
@fanquake, just curious why the trailing comment line here without any comment.  Is this for automated documentation?